website/docs/product/telemetry.md

```diff
@@ -1,65 +1,113 @@
+---
+description: This page provides information on the data collected by Appsmith.
+---
+
 # Telemetry
 
 Telemetry in Appsmith refers to the collection of data about how users interact with the platform. This data helps the Appsmith team identify usage patterns, troubleshoot issues, and make informed decisions about new features and improvements.
 
 ## Data collected by Appsmith
 
-Appsmith is a lightweight proxy and **does not capture** any data returned by your APIs, databases, or third-party tools. All data captured from a self-hosted instance is completely anonymised.
+Appsmith is a lightweight proxy and **does not capture** any data returned by your APIs, databases, or third-party tools. All data captured from a self-hosted instance is completely anonymised for user information.
 
 ### Keep-alive ping
 
-The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used.
+The Appsmith server sends a keep-alive ping every 2 hours to indicate that it's still running without any errors. Additionally, it captures anonymous usage data to track the usage patterns of plugins. This data is collected irrespective of whether telemetry is turned on or off.
```

Comment:
"Additionally, it captures anonymous usage data to track the usage patterns of plugins" - why do we need this it "Keep-alive ping" section, it has nothing to do with it. I'm saying lets just get rid of it